src/controllers/block.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+class Block():
+    def __init__(self, block_columns, block_data):
+        self.block_columns = block_columns
+        self.block_data = block_data
+
+    def to_json(self):
+        block_json = {}
+        col_key = list(self.block_columns)
+        col_data = list(self.block_data)
+        for column_index in range(len(self.block_columns)):
+            block_json[col_key[column_index]] = col_data[column_index]
+        return(block_json)
+    def get_key_values(self):
+        return list(self.block_columns)
+    def save_in_database(self, collections):
+        block = collections.find_one({"x":self.block_data[1],"y":self.block_data[2],"z":self.block_data[3]})
+        if block != None:
+            return
+        collections.insert_one(self.to_json())
+
+    def print_block(self):
+        print(self.to_json())
+
+    def block_mass(self):
+        block_json = self.to_json()
+        mass_name = input("Enter the name of mass column: ")
```

Comment:
This is a failure of separating your program into layers. Block should be a Domain layer class, and it shouldnt have Application layer code such as handling the console input. This will make it harder when we have other ways of accessing the Domain thar is not the console.

Reply:
fixed